lib/digitalocean/bootstrap/tf-default-vars.j2.tf

```diff
@@ -37,7 +31,7 @@ variable "oks_master_size" {
 }
 
 variable "oks_master_node_count" {
-  default = 2
+  default = 4
```

Comment:
oO? Why do we need to setup master nodes? 3 is better than 4. Please choose between 3 or 5 for the quorum vote